apollo-rx2support/build.gradle

```diff
@@ -0,0 +1,58 @@
+buildscript {
+  dependencies {
+    classpath dep.androidPlugin
+  }
+}
+
+apply plugin: 'com.android.library'
+clearSkipApolloRuntimeDep()
+
+android {
```

Comment:
which would also mean converting the rx2 module to pure java? So I'll create an issue for rx1 module and create a PR for it a well.